app/views/booking_requests/step_two.html.erb

```diff
@@ -130,7 +130,7 @@
           <% if @booking_request.errors.include?(:accessibility_requirements) %>
             <span class="error-message"><%= @booking_request.errors[:accessibility_requirements].to_sentence %></span>
           <% end %>
-          <%= f.check_box :accessibility_requirements, class: 't-accessibility-requirements' %> I have accessibility needs
+          <%= f.check_box :accessibility_requirements, class: 't-accessibility-requirements' %> I have accessibility needs <span style="color:#6f777b">eg wheelchair access</span>
```

Comment:
@benbarnett Aman attempted to use a form hint but because it's a block element it was wrapping underneath the checkbox and label - @amansinghb get it fired up on your machine, change it to a form hint and we'll take a look at it together to get it displayed inline.